DataFormats/Provenance/interface/Provenance.h

```diff
@@ -45,6 +45,7 @@ namespace edm {
 
     ProductProvenance const* productProvenance() const;
     BranchID const& branchID() const {return stable().branchID();}
+    BranchID const& originalBranchID() const {return stable().originalBranchID();}
```

Comment:
Please add a comment explaining the difference between `branchID` and `originalBranchID`